x-pack/plugins/security_solution/public/detections/components/rules/rule_actions_overflow/index.test.tsx

```diff
@@ -218,19 +204,18 @@ describe('RuleActionsOverflow', () => {
       wrapper.update();
       wrapper.find('[data-test-subj="rules-details-duplicate-rule"] button').simulate('click');
       wrapper.update();
-      expect(duplicateRulesAction).toHaveBeenCalledWith(
-        [rule],
-        [rule.id],
-        expect.anything(),
-        expect.anything()
+      expect(executeRulesBulkAction).toHaveBeenCalledWith(
+        expect.objectContaining({ action: 'duplicate', search: { ids: ['id'] } })
       );
     });
   });
 
   test('it calls editRuleAction after the rule is duplicated', async () => {
```

Comment:
minor nit: Test titles still using `editRuleAction`/`duplicateRulesAction` which are no longer the functions being called here. Tests still read fine/make sense to me, so no change necessary